feature-group-definitions/anchor-positioning.dist.yml

```diff
@@ -0,0 +1,170 @@
+# Generated from: anchor-positioning.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Anchor positioning
+description: Anchor positioning places an element based on the position of another element. For example, you can place a tooltip next to the content it references.
+spec: https://drafts.csswg.org/css-anchor-position-1/#anchoring
+caniuse: css-anchor-positioning
+status:
+  baseline: false
+  support: {}
```

Comment:
I was expecting to see Chrome 125 here. Are one of the BCD keys not supported in Chrome 125?

Reply:
Ah, of course, it's because it's in beta and not released yet.

Reply:
Ah ok, that's why! I was surprised too, and looked at all of the keys one by one, failing to find the one that wasn't supported.